packages/suite-desktop-core/e2e/tests/general/suite-guide.test.ts

```diff
@@ -1,34 +1,36 @@
 import { test, expect } from '../../support/fixtures';
 
-test.use({ startEmulator: false });
-/**
- * Test case:
- * 1. Go to Bug section in Suite Guide
- * 2. Select Dashboard
- * 3. Write into feedback field
- * 4. Submit bug report (reporttext)
- */
-test('Send a bug report', async ({ suiteGuidePage }) => {
-    await suiteGuidePage.openPanel();
-    await suiteGuidePage.supportAndFeedbackButton.click();
-    await suiteGuidePage.sendBugReport({
-        location: 'account',
-        report: 'Henlo this is testy test writing hangry test user report',
+test.describe('Suite Guide', { tag: '@suite' }, () => {
+    test.use({ startEmulator: false });
```

Comment:
aha, this is some kind of beforeeach

Reply:
Yeah, kind of. `test.use` allow you to configure board range of test properties. Starting from browser type, baseUrl etc. And thanks to our change even how you setup or whether to setup the emulator.

Majority of our test need emulator, and need to get thru the initial Onboarding. We removed the need defining that in every test by moving it to fixtures. Also we are handling there logic web vs desktop and specific needs for the emulator/bridges.
But the test still need to specify params for the emulator/bridges. We handle that thru test.use
I will see how practical it is in large scale and improve based on that. Also any feedback is welcomed.